rpms/SPECS/wazuh-manager.spec

```diff
@@ -298,6 +299,14 @@ if [ $1 = 2 ]; then
   fi
 fi
 
+VD_FILENAME='vd_1.0.0_vd_4.8.0.tar.xz'
```

Comment:
We should find the best way to manager the file versioning.

Reply:
For the moment it will be handled like this.

Reply:
Ok, thank you.